test/testframe.c

```diff
@@ -552,16 +646,7 @@ GetTestNumErrs(void)
 void
 IncTestNumErrs(void)
 {
-    num_errs++;
-}
-
-/*
- * Retrieve the current Test Parameters pointer.
- */
-H5_ATTR_PURE const void *
-GetTestParameters(void)
```

Comment:
Previously, the testing framework had a global pointer that it set to the uncopied test data stored for each test when the test was executing. Then `GetTestParameters()` just returned that pointer to the test. Since the testing framework now copies test data to make it easy to re-use test data between calls to `AddTest()`, this was simplified by passing the stored pointer directly and removing the need to call `GetTestParameters()`.